.github/workflows/monthly.yml

```diff
@@ -0,0 +1,41 @@
+name: monthly
+
+on:
+  push
+    branches:
+      - master
+
+  schedule:
+    - cron: '0 0 1 * *'
+
+jobs:
+  run:
+    name: Refresh month of man pages
+    runs-on: ubuntu-18.04
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Set up Ruby
+        uses: ruby/setup-ruby@v1
+        with:
+          ruby-version: ${{ matrix.ruby }}
+          bundler: none
+
+      - name: Install dependencies
+        run: rake setup
+
+      - name: Update month
+        run: bin/rake man:build
+
+      - name: Commit changes
+        uses: EndBug/add-and-commit@v4
+        with:
+          author_name: Utkarsh Gupta
+          author_email: utkarsh@debian.org
```

Comment:
```suggestion
          author_name: The Bundler Bot
          author_email: bot@bundler.io
```

Reply:
Thanks, @hsbt! :fire: 
Since I am fixing this entire `name` section, so I'll accommodate this in the next commit :smile: